src/modules/wlr/taskbar.cpp

```diff
@@ -539,13 +580,23 @@ Taskbar::Taskbar(const std::string &id, const waybar::Bar &bar, const Json::Valu
 
     /* Get the configured icon theme if specified */
     if (config_["icon-theme"].isString()) {
-        icon_theme_ = Gtk::IconTheme::create();
-        icon_theme_->set_custom_theme(config_["icon-theme"].asString());
-        spdlog::debug("Use custom icon theme: {}.", config_["icon-theme"].asString());
-    } else {
-        spdlog::debug("Use system default icon theme");
-        icon_theme_ = Gtk::IconTheme::get_default();
+        auto icon_theme_config = config_["icon-theme"].asString();
+        size_t start = 0, end = 0;
+
+        do {
```

Comment:
Switch icon-theme to array, and keep `if (config_["icon-theme"].isString()) {` if we pass just one icon theme.
Otherwise, add an `if (config_["icon-theme"].isArray()) {` it'll much simpler than splitting by `,`

Reply:
Thank you, will fix this.